src/external/wb_presets.c

```diff
@@ -6743,6 +6743,18 @@ const wb_data wb_preset[] =
   { "Sony", "ILCE-6000", "6000K", 0, { 2.98828125, 1, 1.546875, 0 } },
   { "Sony", "ILCE-6000", "8500K", 0, { 3.453125, 1, 1.30859375, 0 } },
 
+  { "Sony", "ILCE-6300", Daylight, 0, { 2.6015625, 1, 1.78515625, 0 } },
+  { "Sony", "ILCE-6300", Shade, 0, { 3.05859375, 1, 1.47265625, 0 } },
+  { "Sony", "ILCE-6300", Cloudy, 0, { 2.79296875, 1, 1.63671875, 0 } },
+  { "Sony", "ILCE-6300", Incandescent, 0, { 1.64453125, 1, 3.2265625, 0 } },
+  { "Sony", "ILCE-6300", WarmWhiteFluorescent, 0, { 1.91015625, 1, 3.1796875, 0 } },
+  { "Sony", "ILCE-6300", CoolWhiteFluorescent, 0, { 2.4296875, 1, 2.625, 0 } },
+  { "Sony", "ILCE-6300", DayWhiteFluorescent, 0, { 2.53125, 1, 1.88671875, 0 } },
+  { "Sony", "ILCE-6300", DaylightFluorescent, 0, { 2.8125, 1, 1.7109375, 0 } },
+  { "Sony", "ILCE-6300", Flash, 0, { 2.9140625, 1, 1.63671875, 0 } },
+  { "Sony", "ILCE-6300", Underwater, 0, { 2.8125, 1, 1.875, 0 } },
+  { "Sony", "ILCE-6300", "5500k", 0, { 2.65234375, 1, 1.74609375, 0 } },
```

Comment:
Setting temperature to 5500K in the temperature iop should nowadays result in very close values